.travis.yml

```diff
@@ -3,8 +3,7 @@ language: php
 php:
   - 5.6
   - 7.0
-  - 7.1
-  - 7.2
+  - 7.3
```

Comment:
I would prefer to keep 7.2 over 7.0.

Reply:
why? we need the upper and lower limits of the language per major.
rest in between should be fine based on this.

Reply:
In theory yes, in practice we've found a bunch of problems where type hinting works differently between 7.1 and 7.2.

Reply:
But it would be the same for 7.3 if it changes after another minor. The changes are always linear here, not fluctuating.
7.0 => 7.1 => 7.2 => 7.3
Whatever you see happening between 7.1 and 7.2 is also present in 7.3.
Can you show be one example that would not be true for the case as I outline it?